### DISCUSSION TITLE
捡了个邦布家人们，它想跟我回家

### DISCUSSION THREAD

**Discussion by wx98** [share121/inter-knot#2332] [其他]
![image](https://github.com/user-attachments/assets/41708dbf-2cb8-4404-973b-d5b199730bd9)
## 姐姐我呀~看这个邦布太可怜了，就把它捡回家了。
![image](https://github.com/user-attachments/assets/6f5df540-8985-421c-ac6f-139ab64b79e8)

**Comment by Huanshenxuanlong** [comment]
`这只邦布是狡兔屋的吧，之前找他们做委托的时候看过，小心妮可老大找你`

**Comment by Phaethon-0019** [comment]
啧……这邦布……没记错应该是狡兔屋老大妮可的吧，上次和治安局联合空洞行动的时候见过

**Comment by YuTinMIn** [comment]
妮可：艾米莉安，你在这里干什么？

**Comment by duskbirds** [comment]
艾米莉安肚子前的充电口好可爱<img src="https://img-static.mihoyo.com/communityweb/upload/460474087917ad69a4ff975a3132a0df.png" alt="妮可 哈哈" width="75" >
